nengo_gui/static/components/value.js

```diff
@@ -58,13 +62,12 @@ Nengo.Value.prototype.on_message = function(event) {
     var data = new Float32Array(event.data);
     data = Array.prototype.slice.call(data);
     var size = this.n_lines + 1;
+    /** since multiple data packets can be sent with a single event,
+    make sure to process all the packets */
```

Comment:
Also the second line should probably be indented to make it visually distinct from the next actual code line.